sources/packages/api/src/utilities/string-builder.ts

```diff
@@ -0,0 +1,67 @@
+import * as dayjs from "dayjs";
+
+/**
+ * Allow the construction of a large string piece by
+ * piece with some useful methods to help the
+ * string manipulation.
+ */
+export class StringBuilder {
+  private readonly contents: string[];
```

Comment:
You can initialize variable here like `private readonly contents: string[] = []`

Reply:
Changed!